[2023-06-13T16:32:38.193+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_use_case.task_extract manual__2023-06-13T14:00:51.720302+00:00 [queued]>
[2023-06-13T16:32:38.203+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_use_case.task_extract manual__2023-06-13T14:00:51.720302+00:00 [queued]>
[2023-06-13T16:32:38.203+0200] {taskinstance.py:1308} INFO - Starting attempt 3 of 1
[2023-06-13T16:32:38.229+0200] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): task_extract> on 2023-06-13 14:00:51.720302+00:00
[2023-06-13T16:32:38.236+0200] {standard_task_runner.py:57} INFO - Started process 12275 to run task
[2023-06-13T16:32:38.245+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'airflow_use_case', 'task_extract', 'manual__2023-06-13T14:00:51.720302+00:00', '--job-id', '55', '--raw', '--subdir', '/Users/simba/Documents/Converteo/cas_airflow1/dags/dag_exemple.py', '--cfg-path', '/var/folders/yj/q_cm8stj3td4yz26dm7d78j80000gn/T/tmppvdm3cr4']
[2023-06-13T16:32:38.250+0200] {standard_task_runner.py:85} INFO - Job 55: Subtask task_extract
[2023-06-13T16:32:38.332+0200] {task_command.py:410} INFO - Running <TaskInstance: airflow_use_case.task_extract manual__2023-06-13T14:00:51.720302+00:00 [running]> on host mbp-de-admin-1.home
[2023-06-13T16:32:38.424+0200] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='airflow_use_case' AIRFLOW_CTX_TASK_ID='task_extract' AIRFLOW_CTX_EXECUTION_DATE='2023-06-13T14:00:51.720302+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-13T14:00:51.720302+00:00'
[2023-06-13T16:35:14.285+0200] {local_task_job_runner.py:291} WARNING - State of this instance has been externally set to None. Terminating instance.
[2023-06-13T16:35:14.343+0200] {process_utils.py:131} INFO - Sending 15 to group 12275. PIDs of all processes in the group: [12275]
[2023-06-13T16:35:14.344+0200] {process_utils.py:86} INFO - Sending the signal 15 to group 12275
[2023-06-13T16:36:14.374+0200] {process_utils.py:149} WARNING - process psutil.Process(pid=12275, name='Python', status='running', started='16:32:38') did not respond to SIGTERM. Trying SIGKILL
[2023-06-13T16:36:14.376+0200] {process_utils.py:86} INFO - Sending the signal 9 to group 12275
[2023-06-13T16:36:14.408+0200] {process_utils.py:79} INFO - Process psutil.Process(pid=12275, name='Python', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='16:32:38') (12275) terminated with exit code -9
[2023-06-13T16:36:14.409+0200] {standard_task_runner.py:172} ERROR - Job 55 was killed before it finished (likely due to running out of memory)
